dsc/src/main.rs

```diff
@@ -615,6 +653,10 @@ impl DscInfo {
             port += rs.port_step;
         }
 
+        // Update our region information with what we found in the config file.
```

Comment:
Same question here: should we `bail!` if they don't match?

Reply:
Warnings in 8acd4d6